src/screens/MainScreen/styles.js

```diff
@@ -7,6 +8,13 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     backgroundColor: '#F5FCFF',
```

Comment:
don't forget to use constants color definition

Reply:
this was like this in the react-native-base project. Anyway, it will change in the next releases.